src/features/network-modifications/twoWindingsTransformer/common/TwoWindingsTransformerDialogTabs.tsx

```
@@ -0,0 +1,78 @@
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import { Tab, Tabs } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { getTabIndicatorStyle, getTabStyle } from '../../../parameters/parameters-style';
+import { TwoWindingsTransformerDialogTab } from './twoWindingsTransformer.utils';
+import { FieldConstants } from '../../../../utils';
+import { useWatch } from 'react-hook-form';
```

ESLint

[error] `react-hook-form` import should occur before import of `../../../parameters/parameters-style`

(import-x/order)